src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Dashboard.vue

```diff
@@ -2,11 +2,13 @@
   <v-app>
     <v-navigation-drawer v-model="drawer" floating class="bg-white">
       <v-list>
-        <v-list-item prepend-icon="mdi-account-edit " title="Sandra Adams" subtitle="sandra_a88@gmail.com">
+        <v-list-item prepend-icon="mdi-account-edit " :title="userStore.fullName" subtitle="sandra_a88@gmail.com">
           <template #subtitle>
-            <v-list-item-title><p class="small">test@test.com</p></v-list-item-title>
             <v-list-item-title
```

Comment:
can we sort out the formatting here? why is this broken again?

Reply:
Good question. Trying `"htmlWhitespaceSensitivity": "ignore"` and that does not chop up the html tag into multiple lines.